package.json

```diff
@@ -333,6 +333,7 @@
     "**/lodash": "4.17.15",
     "**/js-yaml": "^3.13.1",
     "**/jpeg-js": "^0.4.0",
+    "**/dot-prop": "^5.1.1",
```

Comment:
Should there also be an accompanying change to `yarn.lock`?

Reply:
Yes, you are right!